Develop1

ArmMart0896 opened the pull request.

Commit:
Выполненное задание Diplom_1 с отчетом JaCoCo

ArmMart0896 committed:
Create README.md

mkouznetsov-git commented on src/test/java/BurgerTest.java:
```diff
@@ -0,0 +1,72 @@
+        burger = Mockito.spy(new Burger());
+        burger.setBuns(bun);
+        burger.getReceipt();
+        Mockito.verify(burger).getReceipt();
```
Не хватает проверки корректности полученного чека

Commit:
Diplom_1 доработанный
Добавлена проверка корректности формирования чека checkCorrectReceiptTest()
Так же обновлен отчет Jacoco
